swagger.yml

```diff
@@ -2259,6 +2279,25 @@ definitions:
         type: string
       error:
         type: boolean
+  pluginRun:
```

Comment:
Should we add `description` to each property?
Adding a `x-example` might be good too?